src/FlipMove.js

```diff
@@ -348,6 +348,14 @@ class FlipMove extends Component<ConvertedProps, FlipMoveState> {
       leavingChildren.forEach(leavingChild => {
         const childData = this.getChildData(getKey(leavingChild));
 
+        // Warn if child is disabled
+        if (
+          !this.isAnimationDisabled(this.props) &&
+          childData.domNode &&
+          childData.domNode.disabled
+        )
+          childIsDisabled();
```

Comment:
Nit: Can we wrap the conditional body in squiggly-brackets? As much as I like the terseness of bracket-free bodies, in my experience it's too easy to introduce subtle bugs. Plus, it's unconventional, and I try to keep with convention unless I have a really good reason to deviate.

I'm surprised Prettier didn't catch and rewrite this... is the hook firing properly?

Reply:
yes, i don't like bracketless (for multi line ifs) either. husky also executes correctly. prettier is cool with it

Reply:
That's because you include `prettier` **preset**, which disables quite a lot of eslint rules, including `curly`:

https://github.com/prettier/eslint-config-prettier/blob/984de70e8c6b57684b444283561019389ccebd11/index.js